### DISCUSSION TITLE
Project Euler

### DISCUSSION THREAD

**Discussion by domix80** [TheAlgorithms/Python#8123] [Ideas]
I suggest to create a Project_Euler where we can keep all the problem resolutions for:
https://projecteuler.net/archives

**Comment by amos-syd** [comment]
Hello @domix80, I think we already have a folder for it: https://github.com/TheAlgorithms/Python/tree/master/project_euler
Feel free to contribute! Thanks

**Comment by domix80** [comment]
Hi @aparibocci,

Thanks so much for letting me know.
